willow/lib/tasks/seed_test_data.rake

```diff
@@ -101,6 +102,7 @@ namespace :willow do
     # finished administrative sets
     ##############################################
 
+    Rake::Task['hyrax:workflow:load'].execute
```

Comment:
Should we replace this line with Hyrax::Workflow::WorkflowImporter.load_workflows as you have done in the feature tests. Looks cleaner.